app/assets/javascripts/admin/utils/filters/unlocalize_currency.js.coffee

```diff
@@ -0,0 +1,8 @@
+angular.module("admin.utils").filter "unlocalizeCurrency", ()->
+  # Convert string to number using injected currency configuration.
+  (price) ->
+    # used decimal separator from currency configuration
+    decimal_separator = I18n.toCurrency(.1, {precision: 1, unit: ''}).substring(1,2)
+    if (decimal_separator == ",")
+      price = price.replace(",", ".")
```

Comment:
What if the price is `$1.000.000,00`?

![Dr. Evil](https://media.giphy.com/media/sEULHciNa7tUQ/giphy.gif)

Seriously, `parseFloat("1.000.000.00") == 1`.

Reply:
Is this case really possible? I mean, can a point (`.`) be a separator of thousands?
Both cases that I can test:
<img width="233" alt="Capture d’écran 2021-05-25 à 09 12 01" src="https://user-images.githubusercontent.com/296452/119454990-7645bf00-bd39-11eb-94a6-50bc7aa9eb73.png">
<img width="269" alt="Capture d’écran 2021-05-25 à 09 12 28" src="https://user-images.githubusercontent.com/296452/119455008-780f8280-bd39-11eb-9d4c-8cf7622a8cf4.png">

Reply:
Anyway, I've handled many cases in my last commit: dd47430f1910530575a1fb8b5566777dee75755f
I think we now cover a large set of cases.

Reply:
> can a point (.) be a separator of thousands?

Yes. Germans use the dot for thousands: https://de.wikipedia.org/wiki/Million